docs/user-guide/ze-usage.md

```diff
@@ -0,0 +1,310 @@
+# Using Zowe Explorer
+
+Use the content of this section to familiarize yourself with the extension, and make the best use of available options and features. The section contains usage tips, sample use cases for data sets, USS files, JOBs, and TSO commands.
+
+## Usage Tips
+
+Use the following tips to familiarize yourself with the extension and make the best use of it:
+
+* **Data set, USS and jobs persistence settings:** You can enable the persistence of any data sets, USS files and jobs by adding them to the **Favorites** tab. Right-click on a data set, USS file or jobs and click **Add Favorite**.
```

Comment:
Worth mentioning what "persistance" means in this context? I'm not sure myself.

Reply:
I would formulate it this way:
"You can store any of data sets, USS files, or jobs permanently in the **Favorites** tab. Right-click on a data set, USS file, or job and click **Add Favorite**." Now, I was trying to use this "add to Favorites" option, and by right-clicking on a profile, one can only add it to Favorites. In this case, the profile is saved with the label 'search' (whatever that means) and a magnifying glass icon next to it. As a user, I would guess it means "a permanently saved search filter". Not sure that's a correct assumption. @zdmullen any ideas?